src/x11-utils.h

```diff
@@ -2,12 +2,14 @@
 #define __HERBST_X11_UTILS_H_
 
 #include <X11/X.h>
+#include <vector>
 
+#include "rectangle.h"
 #include "x11-types.h"
 
-// cut a rect out of the window, s.t. the window has geometry rect and a frame
-// of width framewidth remains
-void window_cut_rect_hole(Window win, int width, int height, int framewidth);
+// cut the specified rectangles out of the window
+void window_cut_rect_hole(Window win, int width, int height,
```

Comment:
Should we update the name to window_cut_rect_holes?

Reply:
I don't care too much :)

Reply:
I've renamed it while  updating the comment.

Reply:
Thank you :)